content/03.text-mining.md

```diff
@@ -18,8 +18,43 @@
 
 #### Unsupervised Machine Learning
 
-1. Mention Clustering Analysis
-2. Mention Co-Occurrence approaches
+Unsupervised machine learning consists of drawing inferences within data without the use of labels.
+Common methods of unsupervised learning involve some form of clustering or statistical calculations.
+In this section we discuss methods that use unsupervised learning to detect relationship asserting sentences from text.
+
+Majority of unsupervised text mining methods use some form of co-occurrence (Table {@tbl:unsupervised-methods-text-mining}), which is an instance of two entities appearing together in text.
```

Comment:
`A majority` or `The majority` probably? It's probably also good to have a reference for this assertion.